ocaml/testsuite/tests/typing-layouts-err-msg/concrete_v1_check.ml

```diff
@@ -0,0 +1,9 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * toplevel
+*)
+
+type t_void : void;;
+(assert false : t_void);;
+(* CR layouts v2.9: This error message is bad, but it should be gone when merged back into
```

Comment:
Why is this going away? It will eventually, but not for months.

Reply:
Just the layout history part will be going away when merged.

The check has been refactor in `main`: 
https://github.com/ocaml-flambda/flambda-backend/blob/177760ad8f2b96538c30db223ec467493a5803b2/ocaml/lambda/translmod.ml#L56-L58
It's not longer creating a sort var just for the error message like before: 
https://github.com/ocaml-flambda/flambda-backend/blob/bc38ff3587c2c6d33f1d3e24f9eea79715849945/ocaml/lambda/translmod.ml#L57-L65
In the future, the error message will just be:
```
Error: Non-value sort void detected in [translmod] in type t_void:
       Please report this error to the Jane Street compilers team.
```
and the reason constructor in question can be deleted.

Reply:
OK -- makes sense. Then perhaps good to leave a CR reminding us to delete the constructor.